ONSdigital/rm-case-service
# Upgrade rm-case-service to java 17

# What and why?
We need to upgrade the service to Java 17 in order to continue to use a compatible and supported version.

# How to test?
Run unit tests

Double check the following endpoints:

1. [CASE_SVC_URL]/cases/[CASE_ID/
2. [CASE_SVC_URL]/cases/[CASE_ID/events
3. [CASE_SVC_URL]/cases/partyid/[PARTY_ID
4. [CASE_SVC_URL]/categories

# Trello
https://trello.com/c/tAHSfIjq/1627-upgrade-rm-case-service-to-java-17
